partiql-plan/src/main/java/org/partiql/plan/Collation.java

```diff
@@ -11,28 +11,27 @@
 public interface Collation {
 
     /**
-     * API WARNING – THIS WILL BE REPLACED WITH AN `int` IN 1.0.
-     * <br>
-     * TODO <a href="https://github.com/partiql/partiql-lang-kotlin/issues/1664">replace with an `int` in 1.0</a>
-     *
-     * @return the column to sort by
```

Comment:
(self-review) sync'd with @johnedquinn on this. Need to keep it as a `Rex` since the specific column may not be known at static time.